src/ol_dbt/models/marts/combined/_marts__combined__models.yml

```
@@ -149,8 +149,8 @@ models:
     description: string, unique string to identify a course run on the corresponding
       platform.
   - name: discount
-    description: str, discount applied for the order in readable format, e.g., $100
-      off, 30% off.
+    description: str, discount applied for the order in readable format, e.g., $100,
+      30%.
```

```suggestion
    description: str, discount applied to the order in a readable format. The value is always an amount taken off the list price of the order, e.g., "$100" means $100 was taken off the price. "30%" means the price was reduced by 30%. 
```

I think this is clearer and more grammatical. Do we use "list price" elsewhere, or is there a better phrase for that?

---

yes, we use `list_price` in marts__mitxpro_ecommerce_productlist. I will update the description. Thanks.